internal/website/src/css/custom.css

```diff
@@ -71,7 +71,7 @@
 
 html,
 body {
-  font-family: 'Open Sans', sans-serif;
```

Comment:
I'm assuming Open Sans is being loaded from a dependency as I couldn't find the source in the codebase. @blakewilson @theodesp @apmatthews any ideas?

Reply:
Interesting 🤔 I don't see it anywhere. I even removed `'Open Sans'` from the `font-family` and it didn't change the font on the docs site, leading me to believe it's not referenced anywhere. Perhaps this was just some code copied over incorrectly when we first spun up the Docs site.